src/javascripts/legacy_app.js

```diff
@@ -41,6 +42,12 @@ var App = {
 
   logClosedModal: function() {
     console.log('Closed a modal.');
+  },
+
+  storeClosedAt: function() {
+    var closedAt = new Date();
+    console.log('CLOSING', closedAt);
+    this.store('lastClosedAt', closedAt);
```

Comment:
Nothing wrong with this PR, was just addressing your earlier comment about where examples should live.